packages/block-editor/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Master
+
+- Add new `__experimentalEditorSkeleton` component. This has been moved over from the `@wordpress/edit-post` package, where it was an internal component called `EditorRegions`. Its class names have thus been renamed from `edit-post-editor-regions` to `block-editor-editor-skeleton`.
```

Comment:
In general we add "sections" to changelog entries. I believe this one could be "### New Features" and the other one could be `### Internal`

Reply:
Ah right, thanks. Done in 67061c600